examples/electron/src-gen/frontend/electron-main.js

```diff
@@ -18,18 +18,28 @@ const electron = require('electron');
 const path = require('path');
 
 let mainWindow = undefined;
+var requireValue;
+
 
 electron.app.on('window-all-closed', function () {
     if (process.platform !== 'darwin') {
         electron.app.quit();
     }
 });
 
+function loadURL() {
+    mainWindow.loadURL(`file://${path.join(__dirname, '../../lib/index.html')}?port=` + requireValue.port);
+}
+
+
 electron.app.on('ready', function () {
-    require("../backend/main");
+
+    requireValue = require("../backend/main")
```

Comment:
Why do you need timeout? Just exporting promise did not work?
`require("../backend/main").default.then(server => {})`